compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -3660,3 +3660,23 @@ bool OMR::Power::CodeGenerator::getSupportsIbyteswap()
    {
    return true;
    }
+
+bool
+OMR::Power::CodeGenerator::supportsNonHelper(TR::SymbolReferenceTable::CommonNonhelperSymbol symbol)
+   {
+   bool result = false;
+
+   switch (symbol)
+      {
+      case TR::SymbolReferenceTable::atomicAddSymbol:
+      case TR::SymbolReferenceTable::atomicFetchAndAddSymbol:
+      case TR::SymbolReferenceTable::atomicSwapSymbol:
+         {
```

Comment:
Does it matter that you missed cases here of data-type-specific symbols, e.g., atomicSwap32BitSymbol?

Reply:
Thanks for raising that question.  I based this on the implementations of `OMR::X86::CodeGenerator::supportsNonHelper` and `OMR::Z::CodeGenerator::supportsNonHelper`.  @0dvictor @fjeremic Victor, Filip - should `supportsNonHelper` include the 32-bit and 64-bit-specific nonhelper symbols?

Reply:
I believe the data-type-specific ones are deprecated and will be deleted soon. AFAIK, none of them is in-use as of now.

Reply:
Thank you!